### PR TITLE
XW-206 | Add tests for Google Form

### PR DESCRIPTION
**Pull request by vforge**
https://wikia-inc.atlassian.net/browse/XW-206

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/GoogleFormWidgetPageObject.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.widget;
+
+import org.apache.commons.lang.NotImplementedException;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/GoogleFormWidgetPageObject.java]: MINOR Remove this unused import 'org.apache.commons.lang.NotImplementedException'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/selenium-tests/blob/6545566e47c09961c4889a94f562837ba8bbb22a/src/test/java/com/wikia/webdriver/testcases/mercurytests/widgettests/AllTagsTests.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/selenium-tests/blob/6545566e47c09961c4889a94f562837ba8bbb22a/src/test/java/com/wikia/webdriver/testcases/widgettests/AllTagsTests.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)

**Comment by tomasznapieralski** [comment]
Please add mercury class to testng

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/GoogleFormWidgetPageObject.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+public class GoogleFormWidgetPageObject extends WidgetPageObject {
+
+    @FindBy(css = "iframe[data-wikia-widget=\"googleform\"]")
+    private WebElement googleFormIframe;
+    @FindBy(css = "div.widget")
+    private WebElement googleFormBody;
+
+    private static final String TAG_NAME = "googleform";
+    private static final String ARTICLE_NAME = "GoogleFormWidget";
+    private static final String TAG = "<googleform url=\"https://docs.google.com/a/wikia-inc.com/forms/d/1cwWn51i5vXFBy7c5VkRzapj6FXxbjZy48VkEZyP33R4/viewform?embedded=true\" />";
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/GoogleFormWidgetPageObject.java]: please respect maximum line length

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/GoogleFormWidgetPageObject.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    private static final String INCORRECT_TAG = "<googleform />";
+    private static final String ERROR_MESSAGE = "Failed to render the Google Form widget. Please check if \"url\" param is properly coped from Embed dialog in Google.";
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/GoogleFormWidgetPageObject.java]: please respect maximum line length

**Comment by tomasznapieralski** [comment]
CR done, few minors

**Comment by WikiaTech** [comment]
SonarQube analysis reported 6 issues:
* MAJOR 6 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/selenium-tests/blob/4b07978168163d5216d35799de31350630dc6b05/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/GoogleFormWidgetPageObject.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/selenium-tests/blob/4b07978168163d5216d35799de31350630dc6b05/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/PollsnackWidgetPageObject.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/selenium-tests/blob/4b07978168163d5216d35799de31350630dc6b05/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/SpotifyWidgetPageObject.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/selenium-tests/blob/4b07978168163d5216d35799de31350630dc6b05/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/TwitterWidgetPageObject.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/selenium-tests/blob/4b07978168163d5216d35799de31350630dc6b05/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/VKWidgetPageObject.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/selenium-tests/blob/4b07978168163d5216d35799de31350630dc6b05/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/WeiboWidgetPageObject.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**